sentry/src/main/java/io/sentry/event/interfaces/SentryStackTraceElement.java

```diff
@@ -13,6 +18,7 @@
     private final Integer colno;
     private final String absPath;
     private final String platform;
+    private final Map<String, Object> vars;
```

Comment:
Ya, good call, I think this all trickled down from the `vars` key name in the actual JSON.